task_manager/task_manager/task_client.py

```diff
@@ -95,7 +95,7 @@ def __init__(
 
         self._goal_handle: Optional[ClientGoalHandle] = None
         self.server_wait_timeout = 10.0
-        self.cancel_task_timeout = 5.0  # Timeout to wait for task to cancel
+        self.cancel_task_timeout = task_specs.cancel_timeout  # Timeout to wait for task to cancel
```

Comment:
Since `task_specs` is saved in the object we could just use that straight in the code instead of copying the timeout data into other member.

Reply:
Done and fixed some tests after this!